exchange/bidder.go

```diff
@@ -173,11 +177,11 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, bidderRequest Bidde
 		dataLen = len(reqData) + len(bidderRequest.BidderStoredResponses)
 		responseChannel = make(chan *httpCallInfo, dataLen)
 		if len(reqData) == 1 {
-			responseChannel <- bidder.doRequest(ctx, reqData[0], reqInfo.BidderRequestStartTime)
+			responseChannel <- bidder.doRequest(ctx, reqData[0], reqInfo.BidderRequestStartTime, reqInfo.TmaxAdjustments)
 		} else {
 			for _, oneReqData := range reqData {
 				go func(data *adapters.RequestData) {
-					responseChannel <- bidder.doRequest(ctx, data, reqInfo.BidderRequestStartTime)
+					responseChannel <- bidder.doRequest(ctx, data, reqInfo.BidderRequestStartTime, reqInfo.TmaxAdjustments)
```

Comment:
Because both `reqInfo.BidderRequestStartTime` and `reqInfo.TmaxAdjustments` are fields of the `reqInfo` struct, should we simplify to simply pass the `reqInfo` instead? It shouldn't be too hard on CPU given that `reqInfo` is an `adapters.ExtraRequestInfo` pointer and these two fields only get read and not modified at all inside `doRequest(..)` so it should be thread safe too.
```
  if len(reqData) == 1 {
    - responseChannel <- bidder.doRequest(ctx, reqData[0], reqInfo.BidderRequestStartTime, reqInfo.TmaxAdjustments)
    + responseChannel <- bidder.doRequest(ctx, reqData[0], reqInfo)
  } else {
```
Thoughts?

Reply:
Fixed here - https://github.com/prebid/prebid-server/pull/2787/commits/2fd4e44b69e5ad091859df73d9bc2b7b55c57e31